### PR TITLE
add MultiClusterService API to api swagger doc

### PR DESCRIPTION
**Pull request by XiShanYongYe-Chang**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

add MultiClusterService API to API swagger doc

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/apis/networking/v1alpha1/service_types.go [hunk 1]
```diff
@@ -5,6 +5,17 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	// ResourceKindMultiClusterService is kind name of MultiClusterService.
+	ResourceKindMultiClusterService = "MultiClusterIngress"
+	// ResourceSingularMultiClusterService is singular name of MultiClusterService.
+	ResourceSingularMultiClusterService = "multiclusteringress"
+	// ResourcePluralMultiClusterService is plural name of MultiClusterService.
+	ResourcePluralMultiClusterService = "multiclusteringresses"
```
Comment (lonelyCZ) [inline pkg/apis/networking/v1alpha1/service_types.go:14]:
```suggestion
	// ResourceKindMultiClusterService is kind name of MultiClusterService.
	ResourceKindMultiClusterService = "MultiClusterService"
	// ResourceSingularMultiClusterService is singular name of MultiClusterService.
	ResourceSingularMultiClusterService = "multiclusterservice"
	// ResourcePluralMultiClusterService is plural name of MultiClusterService.
	ResourcePluralMultiClusterService = "multiclusterservices"
```

Comment (XiShanYongYe-Chang) [reply pkg/apis/networking/v1alpha1/service_types.go:14]: Thanks a lot~

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**